src/data/markdown/docs/03 cloud/03 Manage/06 Audit Trail.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Audit Trail'
+excerpt: 'Monitor actions performed by your organization'
+slug: '/cloud/manage/audit-trail'
+---
+
+The *Audit Trail* can provide valuable insights into the activity taking place within your organization.
+
+> ⭐️ Audit Trail is available on annual Pro and Enterprise plans.
+
+To view the Audit Trail, go to  **Manage > Audit Trail** section.
+
+## What is an Audit trail
```

Comment:
```suggestion
## A list of system events and activities

```

Reply:
Maybe instead of list, we should use "Log" here? I'm thinking that some people may conceive of audit trails as logging, so adding this word somewhere may help searchers find the page.

If this is an inaccurate use of use of the word "log," of course, discard this suggestion.